### PR TITLE
Dhruv sood patch 1

### PR DESCRIPTION
**Pull request by Dhruv-Sood**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8083"></a>

### CODE REVIEW

File: data/Dhruv-Sood.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            "url": "https://www.youtube.com/@SoodDhruv"
+        }
+    ],
+    "tags": [
+        "Open Source",
+        "Javascript",
+        "Typescript",
+        "Python",
+        "React"
+    ]
```
Comment (Vishrut19) [inline data/Dhruv-Sood.json:36]: You need to add `tags` just after `bio`

Comment (Vishrut19) [inline data/Dhruv-Sood.json:36]: You should add `tags` just after `bio` first.

File: data/Dhruv-Sood.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        {
+            "icon": "FaGithub",
+            "url": "https://github.com/Dhruv-Sood"
+        },
+        {
+            "icon": "FaYoutube",
+            "url": "https://www.youtube.com/@SoodDhruv"
+        }
+    ],
+    "tags": [
```
Comment (Vishrut19) [inline data/Dhruv-Sood.json:30]: You should add `tags` above just after the `bio` section.

**Review by Vishrut19 — CHANGES_REQUESTED** [review]

File: data/Dhruv-Sood.json (hunk 2 above)
Comment (eddiejaoude) [reply data/Dhruv-Sood.json:30]: In json the order of the object properties does not matter

Comment (Vishrut19) [reply data/Dhruv-Sood.json]: okay, I saw the template that's why I thought that it is not right. Thanks for helping me with it Eddie.

**Pull request closed without merging**